doc/references.rst

```diff
@@ -50,9 +50,10 @@ If you use the :code:`qualitymetrics` module, i.e. you use the :code:`analyzer.c
 or :code:`compute_quality_metrics()` methods, please include the citations for the :code:`metric_names` that were particularly
 important for your research:
 
-- :code:`amplitude_cutoff` or  :code:`isi_violation` [Hill]_
+- :code:`amplitude_cutoff` [Hill]_
 - :code:`amplitude_median` or :code:`sliding_rp_violation` [IBL]_
```

Comment:
Shouldn't we just move this one too while we at it?

Reply:
Move what where?

Reply:
Sorry the highlighting wasn't clear.... My bad. You moved isi_violation to its own line so I thought just for aesthetics it might be nice to also move the `sliding_rp_violation` to it's own line as well?

Reply:
Ahh, not for aesthetics. Each line was a list of metrics with the appropriate citation. But now there's only one citation with more than 1 metric. Agree that it looks better in two lines if there's one outlier - thanks!